v2/product/products.md

```diff
@@ -0,0 +1,16 @@
+# Products
+
+... needs picture of product admin ...
+
+In the new architecture, a Product entity references one or more ProductVariation entities. Thus the product entities
+replace D7 product display nodes (and match the D8 nodes visually) while the variation entities replace D7 product
```

Comment:
I've been trying to avoid the D7 -> D8 difference. Maybe we find a way to just make a footnote for those moving from D7 to D8?